chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -217,19 +235,23 @@ Notes: Script here
 
 ---
 
-We can plot in the same way as before 
+Now that we have our `sugar_df` in the correct format, we can proceed to plot the mean sugar content for each manufacturer.
```

Comment:
Using Altair we can plot the mfr column which we've identified as a nominal value on the x axis and sugars which we agreed was a quantitative value on the y axis. (Also we can't forget our title!)